docs/src/requirements/child-devices.md

```diff
@@ -0,0 +1,76 @@
+2) Should thin-edge provide routing of data between child devices (from one child device to another)?
+3) Will there be one "child device process" per "physical child device"? Or a single process can manage multiple physical child devices at once?
+4) Should thin-edge also support receiving data from multiple child devices and combining all that data and associate it with the main thin-edge device twin itself in the cloud(instead of having multiple child devices)? Customer typically do this as a cost-cutting measure with clouds charging fees per device.
+
+## Use-Cases
+
+* **Use-Case 1: Gathering telemetry data from connected PLCs and handle as Measurements, Alarms or Events**
+   - a process on the thin-edge device collects process-data from a CODESYS PLC via local network (using the CODESYS PLC handler)
+   - per each connected PLC a process is running
+   - the process pushes gathered PLC's process-data to thin-edge measurements, alarms or events
+   - the process has a configuration file that contains mapping information, meaning which process-data to push to which thin-edge measurement, alarm or event;
+     and which child-device ID to use
+   
+
+* **Use-Case 2: Viewing/applying data-mapping configuration of PLCs using Configuration Management**
+   - the process' configuration (see process configuration file in use-case 1) is managed by Configuration Management
+   - each PLC appears as child-device in the Cloud, that supports individual Configuration Management functionality per child-device
+   - thereby data mapping can be changed or extended from cloud side
+
+* **Use-Case 3: Update application on connected PLCs using Software Management**
+   - each PLC appears as child-device in the Cloud, that supports individual Software Management functionality    
+   - a commandline tool on the thin-edge device is capable to send a CODESYS application (one file) to a connected PLC, and start it
+   - on an incoming SW Management update request for a PLC that commandline tool is invoked to install the CODESYS application on the according PLC
+
+
+NOTE 1: For use-cases above the external device could be also another PLC than CODESYS or even another device than a PLC.
+
+NOTE 2: Use-case(s) with external device that can directly access thin-edge interfaces (e.g. tedge/measurements) not yet covered.
+
+# Requirements
+
+- **Requirement 1:** Thin-edge has to provide child-device support for interfaces: <br/>
+     - Measurements/Alarm/Events<br/>
+     - Configuration Managements<br/>
+     - Software Managements Plugin API<br/>
+
+- **Requirement 2:** Where thin-edge provides child-device support it uses corresponding Clouds' child device meachnism, to associate data/functionality of that child-device as own device underneath the thin-edge device.
+
+- **Requirement 3:** When data is pushed to thin-edge for a child-device, thin-edge will take care that the child-device exists in the Cloud, and creates it if needed.
```

Comment:
Not fully convinced of this requirement yet. It adds unnecessary burden on thin-edge to be stateful(to maintain the list of child devices created already)